### ISSUE TITLE
ObjectMapper Bug kills Messaging

### ISSUE THREAD

**Issue by sboesebeck** [sboesebeck/morphium#76]
The latest version does have a bug in `ChangeStream` processing that kills Messaging altogether.

The issue is in `ObjectMapperImpl.fillList()   (ObjectMapperImpl.java:1368)`  an NPE is thrown as `elementClass` is null. Which is null when de-serializing an List without generic types.

V4.2.0 is _not_ affected - so please downgrade for the time being.

StackTrace:

``` 
de.caluga.morphium.driver.MorphiumDriverException: internal error: java.lang.NullPointerException
	at de.caluga.morphium.driver.mongodb.DriverHelper.handleNetworkError(DriverHelper.java:86)
	at de.caluga.morphium.driver.mongodb.DriverHelper.doCall(DriverHelper.java:41)
	at de.caluga.morphium.driver.mongodb.MongoDriver.watch(MongoDriver.java:989)
	at de.caluga.morphium.changestream.ChangeStreamMonitor.run(ChangeStreamMonitor.java:155)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: java.lang.NullPointerException
	at de.caluga.morphium.ObjectMapperImpl.deserialize(ObjectMapperImpl.java:1194)
	at de.caluga.morphium.changestream.ChangeStreamMonitor$1.incomingData(ChangeStreamMonitor.java:130)
	at de.caluga.morphium.driver.mongodb.MongoDriver.processChangeStreamEvent(MongoDriver.java:981)
	at de.caluga.morphium.driver.mongodb.MongoDriver.lambda$watch$13(MongoDriver.java:1024)
	at de.caluga.morphium.driver.mongodb.DriverHelper.doCall(DriverHelper.java:28)
	... 3 common frames omitted
Caused by: java.lang.NullPointerException: null
	at de.caluga.morphium.ObjectMapperImpl.fillList(ObjectMapperImpl.java:1368)
	at de.caluga.morphium.ObjectMapperImpl.fillMap(ObjectMapperImpl.java:1510)
	at de.caluga.morphium.ObjectMapperImpl.fillMap(ObjectMapperImpl.java:1454)
	at de.caluga.morphium.ObjectMapperImpl.deserialize(ObjectMapperImpl.java:961)
	... 7 common frames omitted
```

**Comment by sboesebeck** [comment]
can be reproduced with the test `MessagingTest.exclusiveMessagingCustomQueueTest()`

**Comment by sboesebeck** [comment]
fixed with 4.2.13
